test/jdk/java/rmi/reliability/benchmark/bench/Makefile

```diff
@@ -47,4 +47,3 @@ all: .classes
 
 clean:
 	rm -f *.class .classes
-
```

Comment:
Hmm, shouldn't this newline at EOF be kept? Asking @ all the people who've reviewed this so far, no need to change it just yet

Reply:
No, it's an extra newline. A file should end with a newline but one is enough.

Reply:
As confusing as they are, unfortunately GitHub UI does not render extra trailing newlines. This is the only one I could find with grepWin.

Reply:
I find the extra trailing newlines through below shell command:
```shell
for i in `find . -iname "Makefile*" | sed "/\.\/build/d"` ; do tail -n 2 $i | grep -c "^$" | grep -q "^1$" ; if [[ 0 -eq $? ]] ; then echo $i ; fi ; done
```

There are only two files has been found:
```
./test/jdk/java/rmi/reliability/benchmark/bench/rmi/Makefile
./test/jdk/java/rmi/reliability/benchmark/bench/Makefile
```

Reply:
Ah, I had not realized that there was more than 1 newline. GitHub's UI confused me here, so we're good to go

Reply:
GitHub's UI assumes the final line has an line break. If it is missing, it displays a red 🚫 at the end of the last line. If there is an empty line showing up in the UI, then it is an additional empty line.

Reply:
In this case, GitHub is actually not showing up extra empty lines, that they appear the same as the single line break in the end of the file.